overlord/devicestate/generate_rsa_key.go

```diff
@@ -0,0 +1,77 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
```

Comment:
This file would be better as _crypto.go_ or similar, so it implies a better scope than this one function.